app/src/main/java/pl/droidsonroids/droidsmap/base/Presenter.kt

```diff
@@ -5,16 +5,21 @@ import pl.droidsonroids.droidsmap.FlowNavigator
 abstract class Presenter<V : MvpView> {
     protected lateinit var view: V
```

Comment:
Tak, ale to jako kolejny feature w TDD, przenieść UiCommandInvoker pod opiekę prezenterów i odpinać widok